lib/components/Navigation/bottom-navigation/bottom-action/bottom-action.scss

```diff
@@ -0,0 +1,42 @@
+.quill-navigation-bottom-bar__action {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    flex-grow: 1;
+    cursor: pointer;
+    height: var(--component-navigationBar-sizing-item-md);
+    border-radius: var(--component-navigationBar-border-radius);
+
+    &-selected {
+        &--true {
+            & > .quill-navigation-bottom-bar__action-icon {
+                svg {
+                    fill: var(--component-navigationBar-label-color-selected);
+                }
+            }
+
+            & > .quill-navigation-bottom-bar__action-label {
+                color: var(--component-navigationBar-label-color-selected);
+            }
+        }
+
+        &--false {
+            & > .quill-navigation-bottom-bar__action-icon {
+                svg {
+                    fill: var(--component-navigationBar-icon-default);
+                }
+            }
+        }
+    }
+
+    &-badge-container {
+        top: var(--core-size-50);
+        right: -9px;
```

Comment:
This is needed because designer are not using the badge container to design the bottom navigation. If we remove this the notification badge will render outside of the line
<img width="79" alt="image" src="https://github.com/user-attachments/assets/ec5aedef-a04f-4a19-9622-a26cb73ed5c2">